README.md

```diff
@@ -1,13 +1,11 @@
 # [![Sarek](docs/images/nf-core_sarek_logo.png "Sarek")](https://nf-co.re/sarek)
 
-> **An open-source analysis pipeline to detect germline or somatic variants from whole genome or targeted sequencing**
```

Comment:
I'm wondering why we lost the description here...
Shall I put it back @ggabernet ?

Reply:
Sure, I thought it was on purpose!